File: src/main/java/com/glass/java/act08/Act08.java
package com.glass.java.act08;

public class Act08 {

    public static void main(String[] args) {

    }

    public static void printAll(String... strings) {
        for (String str : strings) {
            System.out.println(str);
        }
    }

    public void repeat(String str, int num, boolean useNewLine) {
        for(int i = 0; i < num; i++) {
            if(useNewLine){
                System.out.println(str);
            }else{
                System.out.print(str);
            }
        }
    }

    public void repeat(String str, int num) {
        repeat(str, num);
    }

    public void repeat(String str) {
        repeat(str, 3, true);
    }
}

File: src/main/kotlin/com/glass/kotlin/act08/Act08.kt
package com.glass.kotlin.act08

fun main() {


}

fun max(a: Int, b: Int): Int {
    return if(a > b) a else b
}

fun repeat(
    str: String,
    num: Int = 3,
    useNewLine: Boolean = true
) {
    for (i in 1..num){
        if(useNewLine) println(str)
        else print(str)
    }
}